src/main/java/frc/robot/RobotContainer.java

```diff
@@ -207,7 +206,10 @@ private void configureButtonBindings() {
                 R_button6.whileHeld(new CMD_changeSetpoint(shoot, 100));
                 R_trigger.whileHeld(new CMD_ShooterManualRPM(shoot));
                 //L_button2 auto aim and shoot
```

Comment:
Please fix this to say L_button3 or delete it and delete the following commented out line as I believe that AutoShoot is now autoShoot so there is no reason to keep it (and potentially have someone comment it back in)?